defaults/main.yml

```diff
@@ -412,13 +412,10 @@ document_merge_service_oidc_userinfo_endpoint: "{{ camac_auth_idp_url }}/realms/
 document_merge_service_git_repository_url: "https://github.com/adfinis-sygroup/document-merge-service/"
 
 # Document merge service git repository
-document_merge_service_version: "v2.0.0"
+document_merge_service_version: 'v3.0.0'
```

Comment:
Double quotes :wink: Use prettier for yml!

Reply:
Fixed. Will do in the future.